glTFLoader/Interface.cs

```diff
@@ -17,6 +17,13 @@ public static class Interface
         const uint JSON = 0x4E4F534A;
         const uint BIN = 0x004E4942;
 
+        const string EMBEDDEDOCTETSTREAM = "data:application/octet-stream;base64";
```

Comment:
I would recommend adding the trailing slash to this constant so that you are actually checking for the slash instead of ignoring the character using `+ 1`.

Reply:
Okey, I'll do that change.

Btw, detecting embedded images is quite tricky and I'm aware there's room for improvement. For example, I'm sure some people will want to use hardware specific images, like DDS or ASTC , I don't know if there's specific mime types for these formats... in which case, is it possible to find images stored as data:application/octet-stream;base64 ?

Reply:
There have been lots of discussions around this already, but the short answer, at least for core glTF 2.0, is that hardware specific textures are not supported. Thus, you don't have to worry about them for now.

Reply:
Okey then.

About the Interface API, there's a few more methods I would like to add, but I don't know if you think it's too much already...

Usually, when I work with file formats, I try to make everything to work with Streams, certainly, it gets complicated when you have multiple files around. The use case is when you try to load a file from within a ZIP or any other package container (which is quite common to use in videogames)

The idea I had in mind was to replicate the previous methods used to load buffers and images, but using lambdas, so the end user could intercept/inject file accesses to whatever package container is using.

Reply:
Let's do that in a separate change.